Hw17

baldezhnik opened the pull request:
Выполнено

baldezhnik committed:
вспомнил, что неплохо бы и коммит сделать

baldezhnik committed:
выполнено задание

OKurcheuskaya commented on app/src/main/java/com/dsd/homework/HW17Activity.kt:
```diff
@@ -0,0 +1,65 @@
+
+    private fun startCountDawnTimer(timeMillis: Long){
+
+        val tvAct17HW2: TextView = findViewById(R.id.tv_act17_hw2)
```
вынеси инициализацию переменных в отдельный метод

OKurcheuskaya commented on app/src/main/java/com/dsd/homework/HW17Activity.kt:
```diff
@@ -0,0 +1,65 @@
+
+        val btnAct17HW1 : Button = findViewById(R.id.btn_act17_hw1)
+        btnAct17HW1.setOnClickListener {
+            startCountDawnTimer((seek*1000).toLong())
```
обработай случай, если ничего не ввели

baldezhnik committed:
Изменения согласно замечаниям
1. Инициализация переменных выполнено в отдельном методе
2. Ничего не ввести впринципе невозможно, из-за использования seekBar и начальных значений.
Так же в seekBar есть проверка на null.
